### ISSUE TITLE
pipeline description

### ISSUE THREAD

**Issue by avilella** [veidenberg/pline#6]
Hi, I've started experimenting with the pipelines that are generated via 'Add step' and 'Import|Export'.

I managed to create a pipeline with two steps, configure it, give it a name and a description, then 'Export'. It produces a beautiful '.json' file that I can copy in their own folder in 'pipelines' (I presume it doesn't need to have its own folder, as it worked without it, right?).

A small question: where does the pipeline description ({"desc": "description here"}) appear in the menu when loaded? I couldn't see it anywhere obvious or hovering the mouse over the pipeline title. Does it need a special flag somewhere?

See example below:

![image](https://user-images.githubusercontent.com/158007/135986163-2f8118ca-db07-45df-a25e-3a1d3bb57021.png)

The description doesn't immediately appear in the .json, but I can add it just above the pipeline key:

```
{
 "name": "csvtk_cut_freq",
 "pipeline": [
  {
   "plugin": "csvtk_cut",
   "name": "csvtk_cut results",
   "inputs": {
    "f": "foo,bar,xyz"
   }
  },
  {
   "plugin": "csvtk_freq",
   "name": "csvtk_freq results",
   "inputs": {
    "f": "foo,bar",
    "trackName1": "_pipe_",
    "trackName1_sel": "_pipe_"
   }
  }
 ]
}
```

With the description (I took the read mapping pipeline as an example):

```
{
 "name": "csvtk_cut_freq",
 "desc":"Does this and that",
"pipeline": [
  {
   "plugin": "csvtk_cut",
   "name": "csvtk_cut results",
   "inputs": {
    "f": "foo,bar,xyz"
   }
  },
  {
   "plugin": "csvtk_freq",
   "name": "csvtk_freq results",
   "inputs": {
    "f": "foo,bar",
    "trackName1": "_pipe_",
    "trackName1_sel": "_pipe_"
   }
  }
 ]
}
```

Example screenshot of the second json:

![image](https://user-images.githubusercontent.com/158007/135986700-82c760ec-c483-4466-b6bd-9e969897d923.png)

**Comment by veidenberg** [comment]
You are right @avilella, that's how the pipeline JSON should to work. I fixed the export issue (`desc` field now included in the JSON) and added a place in the interface to show the description (info icon next to the pipeline title). Thanks!
